nextflow.config

```diff
@@ -68,25 +63,24 @@ try {
 }
 
 profiles {
-  awsbatch { includeConfig 'conf/awsbatch.config' }
-  conda { process.conda = "$baseDir/environment.yml" }
-  debug { process.beforeScript = 'echo $HOSTNAME' }
+  awsbatch { includeConfig 'conf/awsbatch.config'}
+  conda { process.conda = "$baseDir/environment.yml"}
+  debug { process.beforeScript = 'echo $HOSTNAME'}
   docker {
     docker.enabled = true
   }
   singularity {
     singularity.enabled = true
+    singularity.runOptions = '-B /run/shm:/run/shm'
```

Comment:
```suggestion

```

Reply:
If its SHH specific, please keep it in the SHH profile - in general the Singularity profile shouldn't do it :-)